packages/runtime/src/helpers/functions.ts

```diff
@@ -47,6 +47,10 @@ export const generateFunctions = async (
     await ensureDir(join(functionsDir, functionName))
     await writeFile(join(functionsDir, functionName, `${functionName}.js`), apiHandlerSource)
     await copyFile(bridgeFile, join(functionsDir, functionName, 'bridge.js'))
+    await copyFile(
+      join(__dirname, '..', '..', 'lib', 'templates', 'server.js'),
+      join(functionsDir, functionName, 'server.js'),
```

Comment:
Can we get some comments on why we're copying this file? (This whole functions.ts file could use those tbh hah, but I won't put that on you)

Reply:
yep, i totally agree! i don't know enough about some of the functions in there to comment it all, but i've added comments to `generateFunctions()` in https://github.com/netlify/next-runtime/pull/1950/commits/fd32f38d9d4e1f8c89f99770f82fb312cfab5998